index.html

```diff
@@ -74,8 +74,37 @@ <h2>Search options</h2>
         <section id="header">
             <h2 id="header_date"></h2>
             <h1 id="header_greet"></h1>
+        
         </section>
 
+        <section id="weather">
```

Comment:
you need use `handelbars-template` for rendering data. only if script get correct data from weather api.